pkg/network/protocols/http2/model_linux.go

```diff
@@ -77,18 +77,18 @@ func (tx *EbpfTx) Path(buffer []byte) ([]byte, bool) {
 	if tx.Stream.Is_huffman_encoded {
 		res, err = decodeHTTP2Path(tx.Stream.Request_path, tx.Stream.Path_size)
 		if err != nil {
-			log.Errorf("unable to decode HTTP2 path due to: %s", err)
+			log.Errorf("unable to decode HTTP2 path: due to: %s", string(tx.Stream.Request_path[:tx.Stream.Path_size]), err)
```

Comment:
The buffer is binary, thus you cannot simply convert it into a string. Also you'd like to see the byte-value to be able to debug it later
```suggestion
			log.Errorf("unable to decode HTTP2 path (%#v) due to: %s", tx.Stream.Request_path[:tx.Stream.Path_size], err)
```